wandb/sdk/data_types/base_types/media.py

```diff
@@ -9,6 +9,7 @@
 from wandb import util
 from wandb._globals import _datatypes_callback
 from wandb.sdk.lib import filesystem
+from wandb.util import FilePathStr
```

Comment:
nit: looks like we use `util.FilePathStr` and `FilePathStr` in this file. Would be good to have a single style in this refactor. My preference: import the util module and use `util.FilePathStr`